client/components/community-translator/translated-success.jsx

```diff
@@ -0,0 +1,19 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const TranslatedSuccess = ( { translationUrl } ) => (
+	<div className="community-translator__success">
+		<p>Thanks for contributing!</p>
+		<p>
+			Your translation has been submitted. You can view it on{' '}
+			<a href={ translationUrl } target="_blank">
```

Comment:
I don't know why eslint didn't report this, but in case of `target="_blank"`, we should also add `rel="noopener noreferrer"` for the best safety.

Reply:
Awesome. I completely forgot about the `window.opener` vulnerability 🙇